### DISCUSSION TITLE
Blank Screen

### DISCUSSION THREAD

**Discussion by EEEDOG** [e107inc/e107#5334] [Q&A Installation]
### Question

New install all goes ok, when it completes I just get a blank screen

Traced to this....

class2.php line642
if(!isset($_E107['no_override']))
{
	$override = e107::getSingleton('override');
}

If I comment out //$override = e107::getSingleton('override'); I can get the site up and running until I click news then we are back to a blank screen.

Any suggestions whats up?

### Expected Source

here

**Comment by Jimmi08** [comment]
e107 version, php version? Did you check PHP error log?  You have there some fatal error. 

https://github.com/e107inc/e107/issues/5126

**Comment by Vodhin** [comment]
I have no issues with my configuration:
`e107 Version 2.3.3 running on PHP Version 8.2.22 on an Apache Server`

You should try setting your server to PHP version 7.4 first, then try higher versions until it fails.

**Comment by EEEDOG** [comment]
I guess the newest E107 as I just downloaded it.
Ubuntu 24.04.1 LTS with all updates.  (on Hyper-V virtual server on Windows 11)
PHP Version 8.3.6
10.11.8-MariaDB
Apache/2.4.58 virtual host.  4x4.uk.to/e/

Joomla works no bother.   h4wdc.co.uk

**Comment by Moc** [comment]
Please check your Apache (server) error log. This will provide more details on what is causing the issue on your server configuration.
